pbsmetrics/prometheus/preload.go

```diff
@@ -91,6 +91,13 @@ func preloadLabelValues(m *Metrics) {
 		adapterLabel: adapterValues,
 		actionLabel:  actionValues,
 	})
+
+	//to minimize memory usage, queuedTimeout metric is now supported for video endpoint only
+	//boolean value represents 2 general request statuses: accepted and rejected
+	preloadLabelValuesForHistogram(m.requestsQueueTimer, map[string][]string{
+		requestTypeLabel:   {isVideoLabel},
```

Comment:
The `isVideoLabel` is specific to the `impressions_requests` metric. It works out because that value is coincidentally the same as `ReqTypeVideo`, but please use `ReqTypeVideo` here in case of future refactors.

Reply:
Fixed!